sql/parser/parser_test.go

```diff
@@ -43,3 +44,9 @@ func TestParserMultiStatement(t *testing.T) {
 		})
 	}
 }
+
+func TestParserDivideByZero(t *testing.T) {
```

Comment:
It’s nice to have a reference for an issue-specific test case.

```suggestion
func TestParserDivideByZero(t *testing.T) {
	// See https://github.com/genjidb/genji/issues/268
```